<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fail-Safe N</string>
  </property>
  <property name="title">
   <string>Fail-Safe N</string>
  </property>
  <property name="subTitle">
   <string>Calculate the fail-safe N</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="method_layout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="method_lbl">
       <property name="text">
        <string>Method to use:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="method_comboBox">
       <property name="toolTip">
        <string>The Rosenthal method calculates the number of studies averaging null results that would have to beadded to the given set of observed outcomes to reduce the combined significance level (p-value) toa target alpha level (e.g., .05). The calculation is based on Stoufferâ€™s method to combine p-valuesand is described in Rosenthal (1979).

The Orwin method calculates the number of studies averaging null results that would have to beadded to the given set of observed outcomes to reduce the (unweighted) average effect size to atarget (unweighted) average effect size. The method is described in Orwin (1983).

The Rosenberg method calculates the number of studies averaging null results that would have to beadded to the given set of observed outcomes to reduce significance level (p-value) of the (weighted)average effect size (based on a fixed-effects model) to a target alpha level (e.g., .05). The method isdescribed in Rosenberg (2005).</string>
       </property>
       <item>
        <property name="text">
         <string>Rosenthal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orwin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="fddfdf">Rosenberg</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="alpha_layout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="alpha_lbl">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>alpha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="alphaSpinBox">
       <property name="toolTip">
        <string>target alpha level to use for the Rosenthal and Rosenberg methods.</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="target_layout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="target_lbl">
       <property name="text">
        <string>target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="target_le">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="toolTip">
        <string>target average effect size to use for the Orwin method. If blank, then the targetaverage effect size will be equal to the observed average effect size divided by2.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="digits_layout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="digits_lbl">
       <property name="text">
        <string>digits:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="digits_SpinBox">
       <property name="toolTip">
        <string>an integer specifying the number of decimal places to which the printed resultsshould be rounded.</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
